modules/geo-layers/src/global-grid-layer/global-grid-decoders/h3-decoder.ts

```diff
@@ -0,0 +1,38 @@
+// deck.gl
+// SPDX-License-Identifier: MIT
+// Copyright (c) vis.gl contributors
+
+import {lerp} from '@math.gl/core';
+import {type GlobalGridDecoder} from './global-grid-decoder';
+import {
+  type CoordPair,
+  type H3IndexInput, // Either string or SplitLong
+  cellToBoundary,
+  cellToLatLng,
+  cellsToMultiPolygon,
+  h3IndexToBigInt
+} from '../h3-js-bigint/h3-js-bigint';
+
+/**
+ * "Standardized" API for working with H3 DGGS tokens.
+ * @note Inspired by similar work in math.gl. May replace by an import in the future.
+ */
+export const H3Decoder = {
+  name: 'h3',
+  // See `main/bundle.ts`, installs a check for the H3 library.
+  initialize: () => {},
+  tokenToBigInt: (token: string) => h3IndexToBigInt(token),
+  cellToLngLat: (cellIndex: bigint) => inPlaceReverseLatLng(cellToLatLng(cellIndex)),
```

Comment:
Perhaps I'm getting obsessed with a detail, but the lat/long reversal is bothering me. How about we add a prop to `GlobalGridDecoder` to specify the position format:

`positionFormat: 'XY' | 'YX'`

similar to `PolygonLayer.positionFormat`. Then `cellToLngLat` could be called `cellToPosition`.

`GlobalGridLayer.indexToBounds` is already invoking `flattenPolygon` which could accept the `positionFormat` as a property and perform the swap at zero cost